libmamba/include/mamba/core/context.hpp

```diff
@@ -130,6 +130,7 @@ namespace mamba
         bool experimental_repodata_parsing = true;
         bool experimental_matchspec_parsing = false;
         bool debug = false;
+        bool use_uv = false;
```

Comment:
Note for later: this should be moved in a dedicated structure for external package managers, but let's do that later when we plug a way to support any external package manager.